nav2_behavior_tree/plugins/action/compute_path_to_pose_action.cpp

```diff
@@ -32,8 +32,37 @@ void ComputePathToPoseAction::on_tick()
 {
   getInput("goal", goal_.goal);
   getInput("planner_id", goal_.planner_id);
-  if (getInput("start", goal_.start)) {
-    goal_.use_start = true;
+
+  // if "use_start" is provided try to enforce it (true or false), but we cannot enforce true if
+  // start is not provided
+  bool bb_use_start;
```

Comment:
```suggestion
  goal_.use_start = false;
  bool bb_use_start;
```

If you initialize it to `false`, then you can remove that line in the next 2x `else` statements (and remove the second else statement entirely).